docs/how-to-guides/block-tutorial/applying-styles-with-stylesheets.md

```diff
@@ -197,6 +197,10 @@ Like scripts, you can enqueue your block's styles using the `block.json` file.
 
 Use the `editorStyle` property to a CSS file you want to load in the editor view, and use the `style` property for a CSS file you want to load on the frontend when the block is used.
 
+It is worth noting that, if the editor content is iframed, both of these will
```

Comment:
Do we have any other documentation that could get linked where folks can learn about the iframed editor concept?

Aside, the included code example doesn't do a great job explaining the purpose of defining `editorStyle`. Maybe, we could test whether we can use a selector like `.wp-block-gutenberg-examples-example-02-stylesheets.is-selected` to show that it's directly related to the editing UI.